Python/libraries/recognizers-text/recognizers_text/matcher/abstract_matcher.py

```diff
@@ -16,10 +16,22 @@ def insert(self, value: [str], id: str):
         pass
```

Comment:
Abstract methods should have a `raise NotImplementedError` or something like that, i dont remember the expeptions actual name, but they should rise an exeption